views/js/qtiCreator/widgets/interactions/graphicGapMatchInteraction/Widget.js

```diff
@@ -115,6 +116,65 @@ define([
             _.forEach(interaction.gapImgs, function(gapImg){
                 $gapList.append(gapImg.render());
             });
+        },
+
+        /**
+         * Display warning message in case matchMax is set to 0 (infinite) and pair is higher that 0
+         * @param {String} template
+         */
+        infinityMatchMax: function (template, choiceMatchMax) {
+            const interaction = this.element;
+            const response = interaction.getResponseDeclaration();
+
+            const isInfinitePair = interaction.getNormalMaximum() === false ? true : false;
+            const hotSpotMatchMax = choiceMatchMax && +choiceMatchMax.attr('matchMax');
+
+            let isInfinityMatchMax = isInfinitePair;
+            const checkTemplates = ['hotspot', 'gapImg'].includes(template);
+            if (hotSpotMatchMax && checkTemplates) {
+                isInfinityMatchMax = false;
+            } else
+            if (isInfinitePair && checkTemplates) {
+                const identifier = choiceMatchMax.attr('identifier');
+                isInfinityMatchMax = this.isChoiceInfinitePair(identifier, template, interaction, response);
+            }
+
+            const $panel = response.renderer.getAreaBroker().getPropertyPanelArea();
+            hider.toggle($(`.response-matchmax-info.${template}`, $panel), isInfinityMatchMax);
+        },
+
+        isChoiceInfinitePair: function (identifier, template, interaction, response) {
+            const mapEntries = response.mapEntries;
+            const pairEntry = _.map(mapEntries, function (value, index) {
+                if (index.includes(identifier) && +value > 0) {
+                    return index.replace(identifier, '').trim();
+                }
+                return false;
+            })[0];
+
+
+            const getGapImgs = interaction.getGapImgs();
+            const getChoices = interaction.getChoices();
+            const mapEntriesLength = Object.keys(mapEntries).length;
+
+            if (!pairEntry) {
+                if (Object.keys(getGapImgs).length > mapEntriesLength || Object.keys(getChoices).length > mapEntriesLength) {
+                    return true;
+                }
+                return false;
+            }
+
+            let isInfinitePair = false;
+            if (template === 'hotspot') {
+                isInfinitePair = _.find(getGapImgs, function (gap) {
+                    return +gap.attr('matchMax') === 0 && pairEntry === gap.attr('identifier');
+                })
```

Comment:
**chore:** missing `;`. And this could be done using an arrow function too.

**suggestion:** 
```suggestion
                isInfinitePair = _.find(getGapImgs gap => +gap.attr('matchMax') === 0 && pairEntry === gap.attr('identifier'));
```